libs/agora/wiki/src/test-setup.ts

```diff
@@ -0,0 +1,20 @@
+import 'jest-preset-angular/setup-jest';
+
+import { server } from 'libs/agora/testing/src/lib/mocks/node';
+
+beforeAll(() => {
+  // Enable API mocking before all the tests.
+  server.listen();
+});
+
+afterEach(() => {
+  // Reset the request handlers between each test.
+  // This way the handlers we add on a per-test basis
+  // do not leak to other, irrelevant tests.
+  server.resetHandlers();
+});
+
+afterAll(() => {
+  // Finally, disable API mocking after the tests are done.
+  server.close();
+});
```

Comment:
This block adds the ability to mock the api calls.  Default api behavior is defined in libs/agora/testing...